### PR TITLE
Enable FindBugs for the project

### PR DESCRIPTION
**Pull request by christ66**
This is a continuation of https://github.com/jenkinsci/support-core-plugin/pull/40.

@reviewbybees

### CODE REVIEW

**Comment by stephenc** [comment]
:bee:

**Comment by armfergom** [comment]
LGTM :bee:

**Comment by andresrc** [comment]
:bee:

Great, I was waiting for this to complete https://github.com/jenkinsci/support-core-plugin/pull/53

File: pom.xml [hunk 1]
```diff
@@ -75,7 +75,8 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
+    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
```
Comment (oleg-nenashev) [inline pom.xml]: Makes sense to bump the version to the latest one

Comment (christ66) [inline pom.xml]: Good point. Will upgrade version.

**Pull request merged**